ESP-sc-gway/loraModem.h

```diff
@@ -148,16 +148,28 @@ unsigned long detTime=0;
 // ----------------------------------------------------------------------------
 // Definition of the GPIO pins used by the Gateway for Hallard type boards
 //
-struct pins {
-	uint8_t dio0=15;	// GPIO15 / D8. For the Hallard board shared between DIO0/DIO1/DIO2
-	uint8_t dio1=15;	// GPIO15 / D8. Used for CAD, may or not be shared with DIO0
-	uint8_t dio2=15;	// GPIO15 / D8. Used for frequency hopping, don't care
-	uint8_t ss=16;		// GPIO16 / D0. Select pin connected to GPIO16 / D0
-	uint8_t rst=0;		// GPIO 0 / D3. Reset pin not used	
-	// MISO 12 / D6
-	// MOSI 13 / D7
-	// CLK  14 / D5
-} pins;
+
+// For MH-ET ESP32 MiniKit 
+#if defined (ARDUINO_ARCH_ESP32) || defined(ESP32)
```

Comment:
This will break almost every other ESP32 board.

Reply:
I was using the Gateway for Hallard type boards on the MH-ET ESP32 MiniKit but did not want to change the _PIN_OUT number.